docs/LOCAL_DEVELOPMENT.md

```diff
@@ -12,6 +12,9 @@ The following dependencies must be installed:
 - node.js >= v16.11 and npm >= 8
 - virtualenv
 - Ruby >= 2.6
+- libsnappy-dev/snappy-devel (debian/centos)
+- docker
```

Comment:
which bit is docker for?

Reply:
The build process that is triggered with make deploy uses Docker to build the images.